packages/video_player/video_player/pubspec.yaml

```diff
@@ -22,6 +22,7 @@ flutter:
         default_package: video_player_web
 
 dependencies:
+  collection: ^1.19.0
```

Comment:
This constraint needs to be adjusted as described in the CI failure output.

Reply:
I didn't realize the fix was in the error. Sorry

Reply:
Needs to be version 1.18.0

Reply:
Thank you very much! I didn’t find the error logs initially, but after you mentioned them, I looked further and found them.